src/components/buttonText.tsx

```diff
@@ -1,8 +1,9 @@
 interface ButtonTextProps {
   text: string;
-  color: "Green" | "Yellow";
+  color: "Green" | "Yellow" | "Red";
   name?: string;
   disabled?: boolean;
+  paddingX?: 2 | 4;
```

Comment:
do we really need to fix only this two options ? Having the possibility to add a className would let us to set flexibly which padding we ever want.

Reply:
I'm open to expanding this. I was following the pattern of restricting the values. If things get out of control in the future, let's expand it to injecting classnames?